### PR TITLE
refactor: remove unused

### PR DESCRIPTION
**Pull request by sxzz**

### CODE REVIEW

File: packages/core/tsconfig.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
     "lib": ["esnext"],
     "paths": {
       "~/*": ["src/*"]
-    }
+    },
+    "noUnusedLocals": true
```
Comment (posva) [inline packages/core/tsconfig.json:20]: let's remove this option while the project is still in early stages

File: tsconfig.json [hunk 2]
```diff
@@ -14,7 +14,9 @@
     "paths": {
       "vue-termui": ["packages/core/src/index.ts"],
       "vite-plugin-vue-termui": ["packages/vite-plugin-vue-termui/src/index.ts"]
-    }
+    },
+    "noUnusedLocals": true,
```
Comment (posva) [inline tsconfig.json:18]: same as above

**Review by posva — CHANGES_REQUESTED** [review]
Thanks!

File: packages/core/tsconfig.json (hunk 1 above)
Comment (sxzz) [reply packages/core/tsconfig.json]: Done

**Review by posva — APPROVED** [review]

**Pull request merged**